general/OKRs/README.md

```diff
@@ -0,0 +1,150 @@
+# Objectives and Key Results
+
+These are the company's OKR for 1Q2018.
+
+Status: Key Results need to be added
+
+Note: the fewer key results the better, the sample ones are just to illustrate where, **not** how many.
+
+### Vision:
+
+* Being the leader of AI on Code
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Company Wide:
+
+* Having successful users of source{d}
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+* Being a better company to work at
+  * Key Result 1
+  * Key Result 2
+  * Key Result 3
+
+### Product:
+
+* Launch of hosted playground
```

Comment:
Engine playground?

Reply:
Yes, hosted engine playground

Reply:
👍 Why not clarifying this in the text of the document as well, to avoid the confustion?

Reply:
Agreed, changed on the epic as well.